### DISCUSSION TITLE
Mac Parallels Desktop: docker desktop unexpected wsl error

### DISCUSSION THREAD

**Discussion by Valerii-Pidhurskii** [ptrumpis/snap-camera-server#93] [Q&A] [answered]
What should I do if I receive "docker desktop unexpected wsl error"
in parallels desktop(win 11) on mac with m chip?
<img width="711" alt="Screenshot 2024-01-13 at 07 21 43" src="https://github.com/ptrumpis/snap-camera-server/assets/65318331/5e9bdc99-ab60-4573-837f-a2d105a4002d">

**Comment by ptrumpis** [comment, marked as answer]
It will not work with mac os parallels.
